xla/service/cpu/onednn_convolution.cc

```diff
@@ -208,6 +210,167 @@ CreateOneDnnPrimDesc<dnnl::convolution_forward::primitive_desc>(
       any_res_md, strides, rhs_dilations, pad_left, pad_right, attrs);
 }
 
+void ExecuteOneDnnConvolution(void* result, void* scratch, void** args,
```

Comment:
This is copy-pasting the old `__xla_cpu_runtime_OneDnnConvolution` with minor modifications. I think we should refactor `__xla_cpu_runtime_OneDnnConvolution` instead. Same for the other three `ExecuteOneDnn{Matmul,Layernorm,Softmax}` functions.